build/aks-variables.yml

```diff
@@ -0,0 +1,22 @@
+variables:
+  deploymentName: 'cd-dicom'
+  azureSubscriptionName: '10M Client Demo'
+  azureContainerRegistry: 'cddicom.azurecr.io'
+  dicomServerComposeLocation: 'build/docker/docker-compose.yaml'
+  dicomCastComposeLocation: 'converter/dicom-cast/build/docker/docker-compose.yaml'
+  fhirEndpoint: 'http://cddicom.westus2.cloudapp.azure.com/fhir'
```

Comment:
We should be routing to the local services inside the cluster. Not public dns
example dicomweb.dicom.svc.cluster.local